packages/gitbeaker-core/src/services/Issues.ts

```diff
@@ -47,16 +47,16 @@ export class Issues extends BaseService {
     return RequestHelper.post(this, `projects/${pId}/issues`, options);
   }
 
-  edit(projectId: string | number, issueId: number, options?: BaseRequestOptions) {
+  closedBy(projectId: string | number, issueId: number, options?: BaseRequestOptions) {
     const [pId, iId] = [projectId, issueId].map(encodeURIComponent);
 
-    return RequestHelper.put(this, `projects/${pId}/issues/${iId}`, options);
+    return RequestHelper.get(this, `projects/${pId}/issues/${iId}/closed_by`, options);
   }
 
-  links(projectId: string | number, issueIId: number) {
-    const [pId, iId] = [projectId, issueIId].map(encodeURIComponent);
+  edit(projectId: string | number, issueId: number, options?: BaseRequestOptions) {
```

Comment:
Though you didnt change this, probably worth fixing in this commit.

Could you rename issueId to [issueIid](https://docs.gitlab.com/ee/api/issues.html#edit-issue)

Reply:
Same change for all the new functions too! Just double checked and they all point to the IId and not the Id

Reply:
Yeah I noticed it was not consistent and I considered to do it, but did not want to override my prerogatives. ;-)
If that helps, I'm gonna do it.

Reply:
Thank you!